website/docs/r/storage_account.html.markdown

```diff
@@ -173,6 +181,23 @@ A `container_delete_retention_policy` block supports the following:
 
 * `days` - (Optional) Specifies the number of days that the container should be retained, between `1` and `365` days. Defaults to `7`.
 
+---
+
+A `change_feed` block supports the following:
+
+* `retention_in_days` - (Optional) The duration of changeFeed retention in days, between `1` and `146000` days. A null value indicates an infinite retention of the change feed.
+
+---
+
+`last_access_time_tracking_policy` exports the following:
+
+* `name` - (Optional) The name of the policy. Default to `AccessTimeTracking`. Current possible value is `AccessTimeTracking`.
+
+* `granularity_in_days` - (Optional) The blob object tracking granularity in days, typically how often the blob object should be tracked. Default to `1`. Current possible value is `1`.
+
+* `blob_type` - (Optional) An array of predefined supported blob types. Default to `blockBlob`. Current possible value is `blockBlob`.
```

Comment:
A few things here:

1. This is an exported block, so needs to be moved below
2. Terraform doesn't support arrays, it's lists
3. The descriptions for `name`, `granularity_in_days` and `blob_type` aren't clear what this does - can we improve this?
4. If `blockBlob` is the only supported value, there's no point exposing this field at this point in time?